web/src/pages/Settings/Groups/GroupModal/GroupModal.jsx

```diff
@@ -54,6 +54,23 @@ const GroupModal = ({ group, zones, onClose, del, apply }) => {
                 onChange={(e) => {
                     setGroupZones(e.target.value);
                 }}
+                MenuProps={{
+                    PaperProps: {
+                        style: {
+                            maxHeight: "30vh",
+                            overflowY: "scroll",  // Ensure scrollbar is always visible (except on iOS because apple hates you)
```

Comment:
iOS browsers are hardcoded to make the scrollbar disappear after a short while and the only way around this is to make a custom scrollbar component

I'm not doing that